index.html

```diff
@@ -148,17 +148,48 @@ <h3 class="work-card-project-title">Multi-Post Stories</h3>
           <h3 class="work-card-project-title">Multi-Post Stories</h3>
           <p class="work-card-project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
           <ul>
-            <li class="project-language">css</li> 
+            <li class="project-language">css</li>
             <li class="languages-separation"></li>
-            <li class="project-language">html</li> 
+            <li class="project-language">html</li>
             <li class="languages-separation"></li>
-            <li class="project-language">bootstrap</li> 
+            <li class="project-language">bootstrap</li>
             <li class="languages-separation"></li>
             <li class="project-language">Ruby</li>
           </ul>
           <button type="button" class="project-button">See Project</button>
         </section>
       </section>
+
+      <section id="contact-me">
+        <div class="contact-me-header">
+          <h2 id="contact-me-title">Contact us</h2>
+          <p id="contact-me-description">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+        </div>
+        <form action="https://formspree.io/f/xjvlpwgy" method="post">
+          <input type="text" name="name" id="contact-me-name" maxlength="30" placeholder="Full name" required>
+          <input type="email" name="email" id="contact-me-email" placeholder="Input your email" value="example@email.com" required>
+          <textarea name="message" id="contact-me-message" maxlength="500" placeholder="Enter your message" required></textarea>
+          <button class="project-button" id="submit-button" type="submit">Send Message</button>
+        </form>
```

Comment:
- [ ] Kindly make sure to use flex, to position form elements, as specified in the project requirements.